sbc_compassion/models/correspondence_template.py

```diff
@@ -119,7 +113,6 @@ class CorrespondenceTemplate(models.Model):
         copy=True,
         readonly=False,
     )
-    nber_keypoints = fields.Integer("Number of key points")
```

Comment:
In fact you can also remove all fields above at lines 94-115 (and all their usages)!
You can remove the model `correspondence.lang.checkbox` and all usages (the security accesses too).